### ISSUE TITLE
Installation crashes on macOS Catalina (10.15.5)

### ISSUE THREAD

**Issue by medikoo** [baileyherbert/utimes#1]
Using Node.js v14.7.0 (but same error ocurs on v12):

```
% npm install utimes

> utimes@3.0.3 install /Users/medikoo/Projects/_tests/test-utime/node_modules/utimes
> node-gyp rebuild

  CC(target) Release/obj.target/nothing/../node-addon-api/src/nothing.o
  LIBTOOL-STATIC Release/nothing.a
  CXX(target) Release/obj.target/binding/binding.o
../binding.cc:166:3: error: no matching function for call to 'New'
                Napi::Function::New(env, utimes)
                ^~~~~~~~~~~~~~~~~~~
/Users/medikoo/Projects/_tests/test-utime/node_modules/node-addon-api/napi-inl.h:1886:25: note: candidate function not viable: no known conversion from 'Napi::Env' to 'size_t' (aka 'unsigned long') for 1st argument
inline Object Function::New(size_t argc, const napi_value* args) const {
                        ^
/Users/medikoo/Projects/_tests/test-utime/node_modules/node-addon-api/napi-inl.h:1784:27: note: candidate template ignored: couldn't infer template argument 'Callable'
inline Function Function::New(napi_env env,
                          ^
/Users/medikoo/Projects/_tests/test-utime/node_modules/node-addon-api/napi-inl.h:1807:27: note: candidate function template not viable: requires at least 3 arguments, but 2 were provided
inline Function Function::New(napi_env env,
                          ^
/Users/medikoo/Projects/_tests/test-utime/node_modules/node-addon-api/napi-inl.h:1878:25: note: candidate function not viable: requires single argument 'args', but 2 arguments were provided
inline Object Function::New(const std::initializer_list<napi_value>& args) const {
                        ^
/Users/medikoo/Projects/_tests/test-utime/node_modules/node-addon-api/napi-inl.h:1882:25: note: candidate function not viable: requires single argument 'args', but 2 arguments were provided
inline Object Function::New(const std::vector<napi_value>& args) const {
                        ^
1 error generated.
make: *** [Release/obj.target/binding/binding.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
gyp ERR! System Darwin 19.5.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/medikoo/Projects/_tests/test-utime/node_modules/utimes
gyp ERR! node -v v14.7.0
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
npm WARN enoent ENOENT: no such file or directory, open '/Users/medikoo/Projects/_tests/test-utime/package.json'
npm WARN test-utime No description
npm WARN test-utime No repository field.
npm WARN test-utime No README data
npm WARN test-utime No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! utimes@3.0.3 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the utimes@3.0.3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/medikoo/.npm/_logs/2020-07-30T09_03_34_902Z-debug.log
```

**Comment by baileyherbert** [comment]
This should now be fixed in v3.0.4, please let me know if you still can't get it working. Thanks!

**Comment by medikoo** [comment]
Thank you @baileyherbert ! I confirm it installs now without issues
